solution/ui/regulations/eregs-vite/src/components/policy-repository/PolicyResults.vue

```diff
@@ -1,14 +1,11 @@
-<script setup>
+<script>
```

Comment:
Using two script tags are an officially documented and supported pattern (see https://mokkapps.de/vue-tips/use-two-script-blocks and https://vuejs.org/api/sfc-script-setup.html?ck_subscriber_id=1460864714#usage-alongside-normal-script) for certain situations.  

One such situation: declaring named exports, which I'm doing here to export methods to be used in Vitest unit tests.